scripts/noshake.json

```diff
@@ -302,12 +305,11 @@
   "Mathlib.Lean.Expr.ExtraRecognizers": ["Mathlib.Data.Set.Defs"],
   "Mathlib.Lean.Expr.Basic": ["Batteries.Logic"],
   "Mathlib.Init.Logic": ["Batteries.Logic"],
-  "Mathlib.Logic.Function.Defs": ["Mathlib.Tactic.AdaptationNote"],
```

Comment:
Are these changes wanted?

Reply:
These were made by `lake exe shake --fix`.

Reply:
Meaning that `shake` complains without them?

Reply:
OK, it builds without that line.